node.md

```diff
@@ -0,0 +1,39 @@
+## Draft Node.js Guidelines
+
+### General
+* Use [Standard JS](https://standardjs.com/) to lint your code. A conistent approach to code layout, spacing and formatting to code makes it easier to switch between projects. By [adding it as a dev dependency](https://standardjs.com/index.html#install) it can be easily used in the terminal, [Webstorm](https://blog.jetbrains.com/webstorm/2017/04/using-javascript-standard-style/), and [Visual Studio](https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs).
+* Session state should not be stored on the node app server
+* Avoid blocking the [main event loop and the worker pool](https://nodejs.org/en/docs/guides/dont-block-the-event-loop/). In short "you shouldn't do too much work for any client in any single callback or task." and consider passing CPU intensive tasks off to another service.
+* Prefer await over callbacks and avoid nested callbacks. This is easily done in [Node 8 and above](https://nodejs.org/api/util.html#util_util_promisify_original).
+
+### Versions
```

Comment:
This section should be a standard rather than guidance, i.e.:
- Our standard Node version is the current Active LTS (10.x at the time of writing).
- We don't drop behind the Maintenance LTS (8.x at the time of writing).
- We don't progress beyond Active LTS.